app/views/govuk_publishing_components/components/_search.html.erb

```diff
@@ -2,7 +2,9 @@
   class_name = "gem-c-search--on-white"
   class_name = "gem-c-search--on-govuk-blue" if local_assigns[:on_govuk_blue].eql?(true)
   size ||= ""
+  no_margin ||= false
```

Comment:
**Reviewing commit by commit**

We could implement this using the `shared_helper` code to do margin, see https://github.com/alphagov/govuk_publishing_components/blob/master/lib/govuk_publishing_components/presenters/shared_helper.rb